torch/functional.py

```diff
@@ -71,6 +71,49 @@ def broadcast_tensors(*tensors):
     return _VF.broadcast_tensors(tensors)  # type: ignore
 
 
+def broadcast_shape(*shapes):
+    r"""
+
+    Similar to :func:`broadcast_tensors` but for shapes.
+
+    This is roughly equivalent to
```

Comment:
Is it computing exactly the same result? Why the "roughly" here?
If it does compute the same result, I think you can simply say "Computes the same result as XXX but in a more efficient way as no new Tensor is actually created".

Reply:
I've fixed to say "equvalent" rather than "roughly equivalent". It really is only roughly equivalent because of the edge case of no shapes, but to make them exactly equivalent would require more code and would hurt clarity.